apps/web-giddh/src/app/settings/address-settings/address-settings.component.html

```diff
@@ -116,37 +116,38 @@ <h1>No addresses found within given criteria.</h1>
         </div>
         <!-- branch profile element start -->
         <div class="d-flex flex-column w-100" *ngIf="organizationType === 'BRANCH'">
-            <div class="branch-profile-address-content col-md-12 mr-b15"
-                *ngFor="let address of addresses; index as addressIndex">
-                <div class="single-branch-detail mr-b15">
-                    <h4><i class="icon-branch-icon"></i>{{address?.name}}</h4>
-                    <div class="row">
-                        <div class="col-md-4">
-                            <label>Address:</label>
-                            <p>
-                                {{address.address}}
-                                <ng-container *ngIf="address.taxType && address.taxNumber">
-                                    <br *ngIf="address.address" />
-                                    {{address.taxType}} : {{address.taxNumber}}
-                                </ng-container>
-                            </p>
-                        </div>
-                        <div class="col-md-4 mr-b1">
-                            <label>State:</label>
-                            <p>{{address.stateCode}} - {{address.stateName}}</p>
-                        </div>
-                        <div class="col-md-4">
-                            <div class="set-default">
-                                <p class="no-selected mr-b1 cursor-pointer" hidden><i
-                                        class="icon-no-selected mr-r05"></i>Set
-                                    as Default</p>
-                                <p (click)="handleSetDefaultAddress(address)"
-                                    class="selected-defualt mr-b1 cursor-pointer">
-                                    <i *ngIf="address.isDefault" class="icon-favorite-selected mr-r05"></i>
-                                    <i *ngIf="!address.isDefault" class="icon-no-selected mr-r05"></i>
-                                    Set as Default</p>
-                                <p (click)="showConfirmationModal(address)" class="cursor-pointer">
-                                    <i class="icon-unlink mr-r05"></i>Unlink Address</p>
+            <div *ngFor="let address of addresses; index as addressIndex">
+                <div class="branch-profile-address-content col-md-12 mb-15">
```

Comment:
**branch-profile-address-content** , wrap by new div and changes the position of ngFOR